server/cpho/views/indicators.py

```diff
@@ -48,6 +60,17 @@ class Meta:
     )
 
 
+class UploadForm(forms.Form):
```

Comment:
perhaps move this into an upload_indicators.py module so you can be looser with util functions like deduce dimension